xrpl/transaction/integration/payment_test.go

```diff
@@ -0,0 +1,124 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/Peersyst/xrpl-go/xrpl/rpc"
+	"github.com/Peersyst/xrpl-go/xrpl/testutil/integration"
+	"github.com/Peersyst/xrpl-go/xrpl/transaction"
+	"github.com/Peersyst/xrpl-go/xrpl/transaction/types"
+	"github.com/Peersyst/xrpl-go/xrpl/websocket"
+	"github.com/stretchr/testify/require"
+)
+
+type PaymentTest struct {
+	Name          string
+	Payment       *transaction.Payment
+	ExpectedError string
+}
+
+func TestIntegrationPayment_Websocket(t *testing.T) {
+	env := integration.GetWebsocketEnv(t)
+	client := websocket.NewClient(websocket.NewClientConfig().WithHost(env.Host).WithFaucetProvider(env.FaucetProvider))
```

Comment:
So for my understanding, what `Host` do we use for this test? local, devnet or testnet?